### ISSUE TITLE
What's up with the Mask in the Embedding Layer 

### ISSUE THREAD

**Issue by jfilter** [jfilter/text-classification-keras#4]
Should padding be 0 and unknown 1?

https://keras.io/layers/embeddings/

**Comment by jfilter** [comment]
https://keras.io/layers/embeddings/

**Comment by jfilter** [comment]
Padding should have have the index of the word set to 0. The masking replaces the values with 'masked out' which is then ignored by e.g. the LSTM layer.
